### DISCUSSION TITLE
Running plugin with Vite Serve defaults to dev mode URLs

### DISCUSSION THREAD

**Discussion by xerubia** [aem-vite/vite-aem-plugin#1] [Q&A] [answered]
Hey there, trying to test out this project, and have run into an issue where the proxy server successfully launches but the only content I see is that of if I ran Vite in dev mode alone, however, going to the root "localhost:3000" shows the correct AEM environment, and further examining CRXDE shows that it is correctly proxying my local AEM instance as all the files, folders, and, packages exist. However, if I were to go to 'localhost:3000/' including anything that follows the '/' it would show me the barebones React app that exists in my App.tsx. 

Any pointers?

**Comment by cshawaus** [comment, marked as answer]
Hi @BANK3Y,

This appears to be a small oversight on my part by which I have hard-coded the AEM port to 4602 by default rather than 4502.

https://github.com/aem-vite/vite-aem-plugin/blob/main/src/index.ts#L10

The solution, for now, would be to explicitly pass your AEM port into the plugin configuration.
https://github.com/aem-vite/vite-aem-plugin/blob/main/src/types.ts#L20

```ts
import { viteForAem } from '@aem-vite/vite-aem-plugin';

export default defineConfig(() => ({
  plugins: [
    viteForAem({
      // ...other options
      aem: {
        host: 'localhost'
        port: 4502,
      },
    }),
  ],
}));
```

I'll get this fixed soon and the `aem` property documented in https://www.aemvite.dev/guide/front-end/vite-plugin/#configuration.

Apologies for the delay in replying.
